compiler/rustc_const_eval/messages.ftl

```diff
@@ -403,8 +403,8 @@ const_eval_uninhabited_enum_variant_read =
 const_eval_uninhabited_enum_variant_written =
     writing discriminant of an uninhabited enum variant
 
-const_eval_unmarked_const_fn_exposed = `{$def_path}` cannot be (indirectly) exposed to stable
-    .help = either mark the callee as `#[rustc_const_stable_indirect]`, or the caller as `#[rustc_const_unstable]`
+const_eval_unmarked_const_item_exposed = `{$def_path}` cannot be (indirectly) exposed to stable
+    .help = either mark the item as `#[rustc_const_stable_indirect]`, or the caller as `#[rustc_const_unstable]`
```

Comment:
This is ambiguous about which item should be marked as such -- the caller or the callee.

Reply:
I'll revert it back to callee since a trait can also be called the callee.